icu4c/source/i18n/myancal.h

```diff
@@ -0,0 +1,327 @@
+// © 2016 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+/*
+ ******************************************************************************
+ * Copyright (C) 2003-2013, International Business Machines Corporation
```

Comment:
Probably don't need this 2003-2013 line and others (I assume this is copy and paste from buddhist as you said)